crates/nu-command/tests/commands/network/http/delete.rs

```diff
@@ -57,7 +57,11 @@ fn http_delete_failed_due_to_server_error() {
         .as_str()
     ));
 
-    assert!(actual.err.contains("Bad request (400)"))
+    assert!(
+        actual.err.contains("Bad request (400)"),
+        "unexpected error: {:?}",
+        actual.err
+    )
```

Comment:
While seeing these assertions, I'm wondering if we should change our http errors. But I think this can be done in another PR.